This patch fixes an issue where meta_mpd.py is installed with executable
permissions only. This results in a 'Permission denied' message for
03-rewrite-python-shebang, and a fatal error for xbps-create when processing
the file hash. Fixed upstream in 7ab3daa.

https://github.com/badaix/snapcast/issues/1021

diff --git a/server/CMakeLists.txt b/server/CMakeLists.txt
index e1271b80..25432cd5 100644
--- a/server/CMakeLists.txt
+++ b/server/CMakeLists.txt
@@ -117,6 +117,6 @@ else()
     install(FILES etc/snapserver.conf COMPONENT server DESTINATION ${CMAKE_INSTALL_SYSCONFDIR})
     install(FILES etc/index.html COMPONENT server DESTINATION ${CMAKE_INSTALL_DATADIR}/snapserver)
     install(DIRECTORY etc/snapweb/ DESTINATION ${CMAKE_INSTALL_DATADIR}/snapserver/snapweb)
-    install(FILES etc/plug-ins/meta_mpd.py PERMISSIONS OWNER_EXECUTE GROUP_EXECUTE WORLD_EXECUTE DESTINATION ${CMAKE_INSTALL_DATADIR}/snapserver/plug-ins/)
+    install(FILES etc/plug-ins/meta_mpd.py PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE DESTINATION ${CMAKE_INSTALL_DATADIR}/snapserver/plug-ins/)
     #install(FILES ../debian/snapserver.service DESTINATION ${SYSTEMD_SERVICES_INSTALL_DIR})
 endif()
